recipes/renode-cores/recipe.yaml

```diff
@@ -0,0 +1,89 @@
+context:
+  name: renode-cores
+  version: "1.15.3"
+  python_min: "3.6"
```

Comment:
```suggestion
```
This should removed as we don't use the variable anywhere in the recipe and the oldest supported Python version is 3.9

Reply:
Done